src/main/resources/assets/immersiveengineering/lang/de_de.json

```diff
@@ -1199,30 +1136,28 @@
   "item.immersiveengineering.logic_circuit": "Logikschaltung",
   "item.immersiveengineering.bannerpattern_hammer": "Bannermuster",
   "item.immersiveengineering.bannerpattern_hammer.desc": "Hammer",
+  "item.immersiveengineering.bannerpattern_wirecutter": "Bannermuster",
+  "item.immersiveengineering.bannerpattern_wirecutter.desc": "Kabelschneider",
+  "item.immersiveengineering.bannerpattern_screwdriver": "Bannermuster",
+  "item.immersiveengineering.bannerpattern_screwdriver.desc": "Schraubendreher",
   "item.immersiveengineering.bannerpattern_bevels": "Bannermuster",
   "item.immersiveengineering.bannerpattern_bevels.desc": "Überzug",
   "item.immersiveengineering.bannerpattern_ornate": "Bannermuster",
   "item.immersiveengineering.bannerpattern_ornate.desc": "Verzierter Rahmen",
   "item.immersiveengineering.bannerpattern_treated_wood": "Bannermuster",
   "item.immersiveengineering.bannerpattern_treated_wood.desc": "Holzvertäfelung",
+  "item.immersiveengineering.bannerpattern_warning": "Bannermuster",
   "item.immersiveengineering.bannerpattern_windmill": "Bannermuster",
```

Comment:
Missing the description here, for the warning signs? "Warnschild" should be appropriate.